### PR TITLE
add section about the unikernel HermitCore

### PR DESCRIPTION
**Pull request by stlankes**
- solves issue #33

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-unikernel.org
```
Comment (samoht) [inline CNAME:1]: please don't delete this file

**Comment by stlankes** [comment]
Do I? This was not my intention. Do you need input from my side?

**Comment by samoht** [comment]
Not sure why you closed that pull-request, the changes looked good to me! The only issue was the deletion of the CNAME file which I believe is useful for supporting custom domain on github pages.

**Comment by samoht** [comment]
But maybe I just got lost in a timeloop :-) not sure why your PR just pop-ed in my radar. Sorry for the noise..

**Comment by stlankes** [comment]
No problem, you merged a similar merge request from @justicezyx Consequently, I closed this request.

**Pull request closed without merging**